### PR TITLE
Add Cypress tests for Device Interface Values page

### PR DESCRIPTION
**Pull request by davidebriani**
This PR adds some basic Cypress tests to check the correct rendering of the page for values of a:
- Properties Interface
- Individual Datastream Interface
- Aggregated Datastream Interface

### CODE REVIEW

**Review by Pavinati — APPROVED** [review]

**Comment by Pavinati** [comment]
This looks good to me, rebase with release-1.0 and fix the conficts and we're good to go

File: cypress/fixtures/test.astarte.IndividualObjectInterface.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "type": "double",
+        "explicit_timestamp": true
+      },
+      {
+        "endpoint": "/sensors/%{sensor}/estimated",
+        "type": "double",
+        "explicit_timestamp": true
+      },
+      {
+        "endpoint": "/temperature",
```
Comment (bettio) [inline cypress/fixtures/test.astarte.IndividualObjectInterface.json:20]: This object aggregated interface is ill-formed.

**Review by bettio — CHANGES_REQUESTED** [review]
Please, make sure that interfaces used for test purposes are valid interfaces.

File: cypress/fixtures/test.astarte.IndividualObjectInterface.json (hunk 1 above)
Comment (Pavinati) [reply cypress/fixtures/test.astarte.IndividualObjectInterface.json:20]: this is not object aggregated

**Review by bettio — APPROVED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**